src/gui/lineedit/src/lineedit.h

```diff
@@ -12,26 +12,29 @@
 
 #include <QLineEdit>
 
-QT_BEGIN_NAMESPACE
 class QToolButton;
-QT_END_NAMESPACE
 
 class LineEdit : public QLineEdit
 {
   Q_OBJECT
 
 public:
-  LineEdit(QWidget *parent = 0);
```

Comment:
This is common Qt object/widget declaration (with parent = 0 by default). What's the point in changing it?